config/initializers/address_lookup.rb

```diff
@@ -0,0 +1,8 @@
+EA::AddressLookup.configure do |config|
+  config.address_facade_server  = ENV["ADDRESS_FACADE_TEST_SERVER"]
+  config.address_facade_port    = ENV["ADDRESS_FACADE_TEST_PORT"]
+  config.address_facade_url     = "/address-service/v1/addresses/postcode"
+
+  config.address_facade_client_id =  ENV["ADDRESS_FACADE_CLIENT_ID"]
+  config.address_facade_key       =  ENV["ADDRESS_FACADE_KEY"]
+end
```

Comment:
would you mind if added explicitly the timeout setting too, so its clear its possible to change it where to change it if needed - its something that they might want to tweak
```
config.timeout_in_seconds = 5
```